Engines/Wine/Verbs/adobeair/script.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "adobeair",
+    "id"                         : "adobeair",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
+    "free"                    : false,
```

Comment:
Is it not free?

Reply:
@madoar it's under all rights reserved i didn't read the terms of use yet.. Just beeing extra carefull around Adobe's software

Reply:
Free is like in "free bear" here

Reply:
beer XD

Reply:
Shame :D

Reply:
Gave me a good laugh. It's a cute idea.

Reply:
lel.. Recommends keeping it free=false for now untill it's proven to not affect phoenicis's intended use.

Reply:
Use free=true, really. Or you'll tell users that they have to "pay" for adobe air

Reply:
Commited on demand